firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryRemoteDocumentCache.java

```diff
@@ -83,17 +83,7 @@ public MaybeDocument get(DocumentKey key) {
     return result;
   }
 
-  /** Remove any documents that the delegate reports as not pinned at the given upper bound. */
-  int removeOrphanedDocuments(MemoryLruReferenceDelegate delegate, long upperBound) {
-    int count = 0;
-    ImmutableSortedMap<DocumentKey, MaybeDocument> updated = docs;
-    for (Map.Entry<DocumentKey, MaybeDocument> entry : docs) {
-      if (!delegate.isPinned(entry.getKey(), upperBound)) {
-        updated = updated.remove(entry.getKey());
-        count++;
-      }
-    }
-    docs = updated;
-    return count;
+  ImmutableSortedMap<DocumentKey, MaybeDocument> getDocuments() {
```

Comment:
Nit: Rename to `getAllDocuments()` to match `getAllDocumentsMatchingQuery()`.

Feel free to disagree :)

Reply:
The other one should change to `getDocumentsMatchingQuery()`.

Reply:
That's fine by me. Can you make this change (if you want, in a follow up?). `getDocumentsMatchingQuery()` is already the name we use on Web.